apps/mull-ui/src/app/pages/create-event/location-autocomplete/location-autocomplete-modal.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import LocationAutoCompleteTextbox from './location-autocomplete-textbox';
+import { CustomTextInput } from '../../../components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMapMarkerAlt } from '@fortawesome/free-solid-svg-icons';
+import { MullBackButton } from '../../../components';
+import { EventRestriction, ILocation } from '@mull/types';
+import { FormikConfig } from 'formik';
+
+export interface LocationAutocompleteModalProps {
+  formik: FormikConfig<{
+    activeRestriction: EventRestriction;
+    startDate: any;
+    endDate: any;
+    startTime: string;
+    endTime: string;
+    eventTitle: string;
+    description: string;
+    location: ILocation;
+    imageFile: string;
+  }>;
+}
+
+export default function LocationAutocompleteModal({
+  formik: { touched, setFieldValue, errors, values },
+}) {
+  const [inputValue, setInputValue] = React.useState('');
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleSetValue = (location: ILocation) => {
+    setInputValue(location.title);
+    setFieldValue('location', location);
+    handleClose();
+  };
+
+  return (
+    <React.Fragment>
+      <CustomTextInput
+        title="Location"
+        fieldName="location"
+        value={inputValue || values.location.title}
+        readOnly
+        onClick={handleClickOpen}
+        hasErrors={touched.location && !!errors.location}
+        errorMessage={errors.location}
+        svgIcon={<FontAwesomeIcon icon={faMapMarkerAlt} />}
+      />
+
+      <Dialog fullScreen open={open}>
+        <MullBackButton
+          style={{
+            marginRight: 'auto',
+            marginLeft: '0.2rem',
+            marginTop: '0.8rem',
+            marginBottom: '2rem',
+          }}
+          children={'Edit'}
+          onClick={handleClose}
+          className={'edit'}
+        />
+        <LocationAutoCompleteTextbox handleSetValue={handleSetValue} input={inputValue} />
```

Comment:
I think the textbox and the search results could use some padding on the left and right.

Reply:
Not sure if this was already done by @ys-lin  but this is what it looks like to me now

![image](https://user-images.githubusercontent.com/35787279/104514525-649c2000-55bf-11eb-9a09-3d9f129f90da.png)

Reply:
I think what he means is that the input text should start where the autocomplete text start if that makes sense